Animations/include/animation/Svg.h

```diff
@@ -1,6 +1,7 @@
 #ifndef MATH_ANIM_SVG_OBJECT_H
 #define MATH_ANIM_SVG_OBJECT_H
 #include "core.h"
+#include "renderer/OrthoCamera.h"
```

Comment:
I know this is super nitpicky, but I try to avoid any includes in header files to reduce compile times if possible :)

It looks like the `OrthoCamera` is only being used as a reference below, so you should be able to just forward declare it and you won't need to include it anymore.